packages/uniorg-parse/src/parser.ts

```diff
@@ -754,6 +758,8 @@ class Parser {
   private parseComment(): Comment {
     let valueLines = [];
     this.r.advance(this.r.forceLookingAt(/^[ \t]*# ?/));
+    const begin = this.r.offset();
```

Comment:
❗🐛 `begin` skipped over `# ?`
```suggestion
    const begin = this.r.offset();
    let valueLines = [];
    this.r.advance(this.r.forceLookingAt(/^[ \t]*# ?/));
```